.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
+node_js:
+  - "4"
```

Comment:
node 4? i would add 
node_js:
  - "7"
  - "8"
  - "9"

Reply:
Odd versions are not stable so we shouldn't be using it.
On the other hand Ember adds version 4 as default to work with any version of ember-cli:

> we recommend testing addons with the same minimum supported node version as Ember.
   so that your addon works for all apps